plugin/trino-delta-lake/pom.xml

```diff
@@ -331,23 +331,8 @@
         <dependency>
             <groupId>io.minio</groupId>
             <artifactId>minio</artifactId>
-            <version>6.0.13</version>
+            <version>7.1.4</version>
```

Comment:
"Bump minio to version 7.1.4"

let's extract this commit and have it merged quicker

Reply:
I was thinking the same as well and that's why I extracted it to https://github.com/trinodb/trino/pull/12171